addons/yippee/userscript.js

```diff
@@ -0,0 +1,137 @@
+export default async function ({ addon, msg, console }) {
+
+//Change font family -- fontSelector in the settings --
+
+var allTextElements = document.querySelectorAll('div, p, label, text, span, a, button, h1');
+
+//The cambiaTexto function is called at the end of the document, in a 'mouseover' eventListener, along with the other functions.
+function cambiaTexto() {
+const fontSelector = addon.settings.get('fontSelector');
+
+		for (var i = 0; i < allTextElements.length; i++) {
+			if (fontSelector == "arial") {
+				allTextElements[i].style.fontFamily = "Arial, Helvetica, sans-serif";
+			}
+			else if (fontSelector == "verdana") {
+				allTextElements[i].style.fontFamily = "Verdana, Tahoma, sans-serif";
+			}
+			else if (fontSelector == "comicSans") {
+				allTextElements[i].style.fontFamily = "'Brush Script MT', Cursive";
+			}
+		}
+}
+
+//Change font size -- fontSizer in the settings --
+
+//This function will also be called at the end.
+function cambiaTamaño() {
+	for (var j = 0; j < allTextElements.length; j++) {
+			allTextElements[j].style.fontSize = addon.settings.get('fontSizer')+'px';
+		}
+}
+
+//Change text to Pictograms -- texToPic in the settings --
+
+//This function will also be called at the end.
+function cambiaPictos() {
+	if (addon.settings.get('texToPic') == true) {
+		const spans = document.querySelectorAll('span, .scratchCategoryMenuItemLabel');
+		const words = ['Kodea', 'Editatu', 'Aldagaiak', 'Sentsoreak', "Hasi saioa", 'Tutorialak', 'Eragileak', "Nire blokeak", 'Kontrola', 'Gertaerak', 'Soinua', 'Itxura', 'Berreskuratu', 'Berria', 'Pertsonaia', 'Mugimendua', 'Norabidea', 'Erakutsi', 'Tamaina', 'Eszena', 'Fitxategia', 'Soinuak', 'Tankerak'];
+		const imagenes = [
+			'https://api.arasaac.org/api/pictograms/8508?plural=false&download=false',
```

Comment:
What is this API for? Is there another way to do the same thing without using an external API?

Reply:
I'm not sure, this API is from a spanish website dedicated to handling these kind of visual resources for free. I guess we could just upload them to the addon or something like that, but I thought this would be simpler and cleaner... I can change it though!